exercises/pov/test/Tests.hs

```diff
@@ -1,28 +1,81 @@
-import Test.HUnit (Assertion, (@=?), runTestTT, Test(..), Counts(..))
-import Data.List (sort)
-import System.Exit (ExitCode(..), exitWith)
-import POV(Graph(..), fromPOV, tracePathBetween)
+{-# LANGUAGE TupleSections #-}
 
-exitProperly :: IO Counts -> IO ()
-exitProperly m = do
-  counts <- m
-  exitWith $ if failures counts /= 0 || errors counts /= 0 then ExitFailure 1 else ExitSuccess
+import Data.Foldable     (for_)
+import Data.Function     (on)
+import Data.Tree         (Tree(Node), rootLabel)
+import Data.List         (sort)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
 
-testCase :: String -> Assertion -> Test
-testCase label assertion = TestLabel label (TestCase assertion)
+import POV (fromPOV, tracePathBetween)
 
 main :: IO ()
-main = exitProperly $ runTestTT $ TestList
-       [ TestList ((tracePathTest : untraceableTest : leafToLeaf : reparentingTests) ++ notFoundTests) ]
+main = hspecWith defaultConfig {configFastFail = True} specs
 
-leaf :: a -> Graph a
-leaf v = Graph v []
+specs :: Spec
+specs = describe "pov" $ do
 
-x :: String
-x = "x"
+    -- As of 2016-09-28, there was no reference file
+    -- for the test cases in `exercism/x-common`.
 
-root :: Graph a -> a
-root (Graph r _) = r
+    describe "tracePathBetween" $ do
```

Comment:
interesting to see that even in the original, before this PR, `tracePathBetween` tests came before `fromPOV`.

I would like to ask whether you think it is appropriate to change so that `fromPOV` comes first. The reason is that the README says that we can use the re-rooted tree to find the path. So I imagine that rerooting the tree will be implemented before finding the path. You can see this too in the example solution - the example solution uses `fromPOV` in its implementation of `tracePathBetween`. Since we are stopping at the first failed test, now it seems important to put `fromPOV` first.